starting.md

```diff
@@ -175,6 +180,15 @@ fun main() {
 }
 ```
 
+**Shell**
+```sh
+curl "https://github.com/Blatzar/scraping-tutorial" | tr -d '\n' |
+  sed -nE "s/.*<article class=\"markdown-body.*<\/path><\/svg>(.*)<\/article>.*/\1/p" # we use the tr -d '\n' command to delete all new lines from the input, since sed works on a line by line basis
+```
+*Note*: 
+Although there are external libraries which can be used to parse html in shellscripts, such as htmlq and pup, these are often slower at parsing than sed (a built-in stream editor command on unix systems).
+This is why using sed with the extended regex flag `-E` is a preferrable way of parsing scraped data when writing shellscripts.
```

Comment:
While good writing, it feels like jumpscare regex before the regex section (in which you have a similar example), how about just showing the notice html parsing and saying that the shell scraping will come below in the regex section.